tool/util/lifecycle.dart

```diff
@@ -1,7 +1,33 @@
 part of dsa.links.repository.util;
 
-fail(String msg, {String out}) async {
-  print("ERROR: ${msg}");
+dbg(String msg) {
```

Comment:
Called to send debug messages. Ignored if `--debug` is not specified on the command line.